src/scripts/make_palette.js

```diff
@@ -25,11 +25,15 @@ function hexToRgb(hex) {
 // var sunshine = ["#FFDAA8", "#FFAA2C", "#FF9218", "#311400"];
 // var noir = ["#E6E6E6", "#979797", "#8A8A8A", "#070707"];
 // var allColors = [alpine, tidal, autumn];
-var tropicana = ["#FFC634", "#7534FF", "#FF34D2"];
```

Comment:
cool I see you were able to decipher this? This is just a script I did to generate the palette quickly